django/contrib/postgres/fields/hstore.py

```diff
@@ -51,6 +52,19 @@ def formfield(self, **kwargs):
         defaults.update(kwargs)
         return super(HStoreField, self).formfield(**defaults)
 
+    def get_prep_value(self, value):
+        value = super(HStoreField, self).get_prep_value(value)
+
+        if isinstance(value, dict):
```

Comment:
@mjtamlyn actually, what about this case:

     HStoreModel.objects.get(field__has_keys=[2, 'a'])

I should probably make it convert list elements to strings too?